docs/howto/use-alternative-secrets-backend.rst

```diff
@@ -429,6 +429,39 @@ When ``gcp_key_path`` is not provided, it will use the Application Default Crede
 The value of the Secrets Manager secret id must be the :ref:`connection URI representation <generating_connection_uri>`
 of the connection object.
 
+Azure Key Vault Backend
+^^^^^^^^^^^^^^^^^^^^^^^
+
+To enable the Azure Key Vault as secrets backend, specify :py:class:`~airflow.providers.microsoft.azure.secrets.azure_key_vault.AzureKeyVaultBackend`
+as the ``backend`` in  ``[secrets]`` section of ``airflow.cfg``.
+
+Here is a sample configuration:
+
+.. code-block:: ini
+
+    [secrets]
+    backend = airflow.providers.microsoft.azure.secrets.azure_key_vault.AzureKeyVaultBackend
+    backend_kwargs = {"vault_url": "https://example-akv-resource-name.vault.azure.net/", "connections_prefix": "airflow-connections", "variables_prefix": "airflow-variables"}
```

Comment:
Same as I commented in the python file, `connections_prefix` and `variables_prefix` should go before `vault_url` to be consistent with the rest of the secrets backend

Reply:
Nice eye for detail there!